File: stara/2012/c++/main.cpp
#include <fstream>
#include "zad4_a.cpp"
#include "zad4_b.cpp"
using namespace std;

int main()
{
    cout << zad4_a() << endl << endl << zad4_b();

    ofstream result("wyniki.txt");

    result << zad4_a() << endl << endl << zad4_b();
}

File: stara/2012/python/main.py
import zad4_a
import zad4_b

# zebranie odpowiedzi z podpunktow i zapisanie ich do pliku
result = '\n'.join([zad4_a.answer, zad4_b.answer])
with open('wyniki.txt', mode='w') as f:
    f.write(result)
